### ISSUE TITLE
After installing "preact-emotion" for a Preact project I'm required to install "prop-types"

### ISSUE THREAD

**Issue by carpben** [emotion-js/emotion#1001]
TLDR: "prop-types" is required for a Preact project with emotion-preact. However this requirement is not mentioned in the docs. 

emotion: 9.2.12,
preact: 8.3.1,
preact-emotion: 9.2.12,
react-scripts-ts: 3.1.0,

What you did:
small CreateReactApp with typescript ptoject. Was imediately converted to Preact. Worked well. Then I installed emotion and preact-emotion. 

What happened:
When running "npm start" the following error showed: 
```
./node_modules/create-emotion-styled/dist/index.esm.js
Module not found: Can't resolve 'prop-types' in 'C:\Users\carpb\projects\gw\gw-widget\node_modules\create-emotion-styled\dist' 
```

Reproduction:
If this isn't straightforward I'll prepare a full reproduction.

Suggested solution:
"npm i prop-types" solved the problem. However it is not mentioned in the docs that "preact-emotion" requires also the installation of "prop-types"

**Comment by gevgeny** [comment]
look like preact is not supported anymore https://github.com/emotion-js/emotion/pull/896

**Comment by Andarist** [comment]
Yes - preact integration got removed from the core, but it should be fairly easy to build your own. Unfortunately, we don't have time resources to handle all possible integrations and we focus on React and Vanilla emotion. Hope you understand.
